apps/services/license-api/src/app/modules/license/guards/licenseTypeScope.guard.ts

```diff
@@ -1,40 +1,43 @@
 import { CanActivate, ExecutionContext, Injectable } from '@nestjs/common'
 import { Observable } from 'rxjs'
 import { Reflector } from '@nestjs/core'
-import { SCOPES_KEY, getRequest } from '@island.is/auth-nest-tools'
+import { getRequest } from '@island.is/auth-nest-tools'
+import { LicenseApiScope, getLicenseTypeScopes } from '@island.is/auth/scopes'
+import { licenseTypeToScope } from '../scopeMapper'
+import { LicenseId } from '../license.types'
 
 @Injectable()
 export class LicenseTypeScopesGuard implements CanActivate {
-  constructor(private readonly reflector: Reflector) {}
-
   canActivate(
     context: ExecutionContext,
   ): boolean | Promise<boolean> | Observable<boolean> {
-    const scopes = this.reflector.getAllAndOverride<string[]>(SCOPES_KEY, [
-      context.getHandler(),
-      context.getClass(),
-    ])
-
     const request = getRequest(context)
 
-    if (scopes && !this.hasScope(scopes, request.auth?.scope)) {
+    const licenseId = request.params.licenseId
+    const isVerify = licenseId === 'verify'
```

Comment:
Afaics this variable is always false.

The 'verify' endpoint does not use a licenseId url param. It also doesn´t need any license type scope logic so it should be just a simple ScopeGuard + Scopes endpoint. That way we can remove all `isVerify ?` logic from this guard.